SCons/Tool/ninja/Methods.py

```diff
@@ -137,6 +137,9 @@ def get_command(env, node, action):  # pylint: disable=too-many-branches
     comstr = get_comstr(sub_env, action, tlist, slist)
     if not comstr:
         return None
+    
+    if isinstance(comstr, SCons.Util.CLVar):
+        comstr = ' '.join(sub_env.subst(comstr, target=tlist, source=slist))
```

Comment:
isn't there a built in way to make a CLVar into a str?

I tried just str(comstr), but the CLVar contained a File node in the list and it was causing exception.

Reply:
It has a `__str__` method, but perhaps it's too simpleminded for that case?
```python
    def __str__(self):
        return ' '.join(self.data)
```

Reply:
yeah that's where the exception happened. join doesn't know what a `File` object is.

Reply:
Maybe try: 
```python
    return ' '.join((str(d) for d in self.data))
```

Reply:
done